packages/builder/src/builder.js

```diff
@@ -1,22 +1,23 @@
 import path from 'path'
 import fs from 'fs'
-
 import pify from 'pify'
```

Comment:
@pi0 Can we replace it with `utils.promisify` or do we need it for the legacy build? 🤔

Reply:
I'm on it ;)

Reply:
@manniL Just created and published [pify-native](https://github.com/blindmedia/pify-native). But it seems that `pify` is way faster. So let's keep using it for now :) (https://github.com/sindresorhus/pify/issues/41#issuecomment-429988506)

Reply:
Alright, thanks for the info @pi0!